### DISCUSSION TITLE
Nodemailler is failing on versel

### DISCUSSION THREAD

**Discussion by HarishankarVj12** [vercel/next.js#52628] [Help]
### Summary

we created a serverless function to send mail using nodemailer, but this is working local and also aws server but not working when deployed to Vercel

### Example

expand.network

**Comment by icyJoseph** [comment]
What errors are you seeing/getting? Deploying a Pages Router, or App Router project?

**Comment by HarishankarVj12** [comment]
App router project, it says something like.
Error: Client network socket disconnected before secure TLS connection was established
    at connResetException (node:internal/errors:717:14)
    at TLSSocket.onConnectEnd (node:_tls_wrap:1595:19)
    at TLSSocket.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'ESOCKET',
  path: undefined,
  host: '172.253.62.109',
  port: 465,
  localAddress: undefined,
  command: 'CONN'
}

**Reply by icyJoseph** [reply]
Is it using Node 18? Try switching between 16 and 18 on the Vercel Dashboard -

**Comment by HarishankarVj12** [comment]
How to change into vercel Dashboard

**Reply by icyJoseph** [reply]
https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings

**Comment by HarishankarVj12** [comment]
No I am facing same issues

**Comment by icyJoseph** [comment]
Alright, just one more thing:

```js
nodemailer.createTransport({
  pool: true,
  host: "smtp.example.com",
  port: 465,
  secure: true, // use TLS ->> 
  auth: {
    user: "username",
    pass: "password",
  },
});
```

Are you using a secure connection? (secure: true) - 

Also, are you awaiting the email transport? or are you just queuing it and then returning?
